src/mixins/utils.repository-mixin.ts

```diff
@@ -0,0 +1,122 @@
+import { MixinTarget } from '@loopback/core';
+import { CrudRepository, Model } from '@loopback/repository';
+import expect from 'expect-runtime';
+import { buildFilterQuery } from '../js/buildFilterQuery';
```

Comment:
Can you try to clear down the linter warnings, or supress them where appropriate?